src/Components/Timer/Timer.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import Timer from 'react-compound-timer';
+import style from './Timer.module.scss';
+
+// const COLOR_CODES = {
+//     info: {
+//         color: '#F4D2D0',
+//     }
+// }
+
+// let remainingPathColor = COLOR_CODES.info.color
+
+const CountdownTimer = (props) => {
+    // const calculateTimeFraction = () => {
+    //     let timeLeft = this.context.value.m * 60 + this.context.value * 1000
+    //     return timeLeft / props.initTime
+    // }
+
+    // const setCircleDashArray =() => {
+    //     const circleDashArray = `${(
+    //         calculateTimeFraction() * 283).toFixed(0)} 283`;
+    //         document.getElementById('base-timer-path-remaining').setAttribute('stroke-dasharray', circleDashArray)
+    // }
+
+    return (
+        <div className={style.countdownTimer}>
+            <Timer 
+                direction='backward'
+                initialTime={props.initTime}
+                lastUnit='m'
+                startImmediately={false}
+                formatValue={(value) => `${(value < 10? `0${value}` : value)}`}
+            >
```

Comment:
Do we want to have a `0` in front of minutes if it is a single digit, or just have it read like `5:32`?

Reply:
I was torn on this! I settled on this for consistency- though I think I like the visual of `5:32` better. I believe we can move this `formatValue` property to the `Timer.Seconds` component in our render if we do want to head this direction.